shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -261,5 +261,85 @@ TEST_F(EmbedderTest, IsolateServiceIdSent) {
   kill_latch.Wait();
 }
 
+TEST_F(EmbedderTest, ShouldBeAbleToCreateAndCollectCallbacks) {
```

Comment:
Should this test be called CanCreateAndCollectcallbacks, for consistency with the test PlatformMessagesCanReceiveResponse  below?

Reply:
Done.